sql/load_demo_data.sql

```diff
@@ -2406,7 +2514,9 @@ CREATE TABLE `oh_malnutritioncontrol` (
   `MLN_LAST_MODIFIED_BY` varchar(50) DEFAULT NULL,
   `MLN_LAST_MODIFIED_DATE` datetime DEFAULT NULL,
   `MLN_ACTIVE` tinyint(1) NOT NULL DEFAULT 1,
-  PRIMARY KEY (`MLN_ID`)
+  PRIMARY KEY (`MLN_ID`),
+  KEY `FK_MALNUTRITIONCONTROL_CREATED_BY_IDX` (`MLN_CREATED_BY`),
+  KEY `FK_MALNUTRITIONCONTROL_LAST_MODIFIED_BY_IDX` (`MLN_LAST_MODIFIED_BY`)
```

Comment:
The demo data version is missing two constraints:
<pre>
  CONSTRAINT `FK_MALNUTRITIONCONTROL_CREATED_BY` FOREIGN KEY (`MLN_CREATED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `FK_MALNUTRITIONCONTROL_LAST_MODIFIED_BY` FOREIGN KEY (`MLN_LAST_MODIFIED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION
</pre>

Reply:
In addition the load  of demo data has the table as MyISAM where the multiple step version has it as InnoDB.

Reply:
Close!
Load demo data:
![image](https://github.com/informatici/openhospital-core/assets/1105445/5cb5a3ac-7a62-4247-9a8c-951b09c51b70)

Load step 04:
![image](https://github.com/informatici/openhospital-core/assets/1105445/d92faa5b-e22d-4a1e-b029-3546a7ca6f80)

Reply:
could you past here the missing lines please?